src/core/oipd-api/daemonApi.js

```diff
@@ -97,7 +97,7 @@ class DaemonApi {
    * let oipd = new DaemonApi("localhost:1606") //leave blank for default API URL
    * let latestArtifacts = await oipd.getLatestArtifacts()
    * ```
-   * @param  {string} [daemonUrl="https://api.oip.io/oip"] - The URL of an OIP Daemon
+   * @param  {Object} [daemonUrl="https://api.oip.io/oip"] - The URL of an OIP Daemon
```

Comment:
might want to update default jsdoc example to match the object

Reply:
Fixed in latest commit :)